grouper/background.py

```diff
@@ -34,19 +35,20 @@ def capture_exception(self):
 
     def run(self):
         while True:
-            logging.debug("Sending emails...")
             try:
                 session = Session()
+                logging.debug("Sending emails...")
                 process_async_emails(self.settings, session, datetime.utcnow())
+                logging.debug("Expiring edges....")
+                expire_edges(session)
+                logging.debug("Pruning old traces....")
                 prune_old_traces(session)
                 session.commit()
                 session.close()
             except OperationalError:
                 Session.configure(bind=get_db_engine(get_database_url(self.settings)))
                 logging.critical("Failed to connect to database.")
-                self.capture_exception()
```

Comment:
this duplication is intentional so we can log OperationalError to sentry as well